CONTRIBUTING.md

```diff
@@ -83,7 +83,7 @@ When you're finished with the changes, create a pull request, also known as a PR
   [git tutorial](https://github.com/skills/resolve-merge-conflicts) to help you
   resolve merge conflicts and other issues.
 
-#### Your PR is Merged!
+#### Your PR is Merged
```

Comment:
See https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md026.md

Reply:
This change kills the enthusiasm we have for merging PRs. We haven't adopted markdownlint as a standard, and I would want to review it and its ruleset before applying its recommendations.